src/main/java/umaikaze/duke/TaskList.java

```diff
@@ -0,0 +1,97 @@
+package umaikaze.duke;
+
+import umaikaze.duke.task.Deadline;
+import umaikaze.duke.task.Event;
+import umaikaze.duke.task.Task;
+import umaikaze.duke.task.Todo;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TaskList {
+    List<Task> list;
+
+    public TaskList() {
+        list = new ArrayList<>(100);
```

Comment:
One of `ArrayList`'s advantage is that it does not have fixed size, unlike `array`. Therefore, you don't necessarily need the `100` size.